src/Model/Release/LibraryRelease.php

```diff
@@ -349,9 +347,9 @@ public function setVersion($version)
     /**
      * True if the current branch ends in the release branch suffix
      */
-    public function isOnReleaseBranch(): bool
+    public function isOnMinorBranch(): bool
```

Comment:
We should probably implement this as `isOnMinorReleaseBranch()` or similarly named instead (i.e. it's on _the correct_ minor branch) and use the plan to determine the correct branch.

Though if you think that's more effort than it's worth I will defer to your judgment.

Reply:
I don't think it's worth it at this stage, I've run simulated (DEV_MODE=1) releases for 4.13.0 and 5.0.0-beta1 and thing seemed to work as they should.

Reply:
This suggestion is more to protect against _human_ error (e.g. I manually checkout `4.12` after resolving merge conflicts, but I should have typed `4.13`)

Reply:
Updated